config/i18n/locales/swahili/translations.json

```diff
@@ -1,67 +1,67 @@
 {
   "buttons": {
-    "forum": "Forum",
-    "donate": "Donate",
-    "load-more-articles": "Load More Articles"
+    "forum": "Jukwaa",
+    "donate": "Changia",
+    "load-more-articles": "Onyesha Makala Zaidi"
   },
   "search": {
-    "label": "Search",
-    "accessible-name": "Submit your search query",
-    "placeholder": "Search 8,000+ tutorials",
-    "see-results": "See all results for {{ searchQuery }}",
-    "no-tutorials": "No tutorials found"
+    "label": "Tafuta",
+    "accessible-name": "Wasilisha swali lako la utafutaji",
+    "placeholder": "Tafuta kati ya mafunzo zaidi ya 8,000+",
+    "see-results": "Tazama matokeo yote ya {{ searchQuery }}",
+    "no-tutorials": "Hakuna mafunzo yaliyopatikana"
   },
   "banner": {
     "default": "Learn to code — {{ <0> }}free 3,000-hour curriculum{{ </0> }}",
```

Comment:
@larymak, this string hasn't been translated yet.

Would you like to translate the current string, and have the link go to [English Learn]()?

Or would you like to adapt the string to read "Chat with other developers in Swahili", and have the link point to the [Swahili chatroom on Discord](https://discord.com/channels/692816967895220344/986667820928602162)?

Reply:
I might have missed a step there, creating a PR after updating the changes, let me do that now